atomic_reactor/constants.py

```diff
@@ -70,3 +70,16 @@
 PLUGIN_PULP_TAG_KEY = 'pulp_tag'
 PLUGIN_ADD_FILESYSTEM_KEY = 'add_filesystem'
 PLUGIN_FETCH_WORKER_METADATA_KEY = 'fetch_worker_metadata'
+
+# max retries for docker requests
+DOCKER_MAX_RETRIES = 3
+# how many seconds should wait before another try of docker request
+DOCKER_BACKOFF_FACTOR = 5
+# docker client retries statuses
+DOCKER_CLIENT_STATUS_RETRY = [404, 408, 500, 502, 503, 504]
```

Comment:
404 should be retried only in some cases, e.g. pulling from registry, which is not related to docker